### DISCUSSION TITLE
Logo does not show up in mails

### DISCUSSION THREAD

**Discussion by cHewN** [requarks/wiki#3242] [Help / Questions] [answered]
**Describe the problem**
When sending a test mail or resetting a password we are getting broken images for the logo.

**To Reproduce**
_Steps to reproduce the behavior:_

- Go to admin panel and use the logo provided below
- Make sure the image has public access
- Fill in the mail settings (we are using office smtp)
- Click on 'Send test mail'
- See broken image

**Expected behavior**
There is a logo in the header of mail.

**Screenshots**
_If applicable, add screenshots to help explain your problem._

Logo in question:
![logo_geoportaal](https://user-images.githubusercontent.com/36958022/111761021-b0a96080-889f-11eb-9254-a3d99bb3a210.jpg)

Email example:
![brokenimage](https://user-images.githubusercontent.com/36958022/111761034-b43ce780-889f-11eb-8c1e-ce2037340182.png)

**Host Info**
- OS: Docker Container (Linux)
- Wiki.js version: 2.5.117
- Database engine: PostgreSQL 11.9
- Node version: 14.9.0

**Additional context**
_Add any other context about the problem here._

**Comment by NGPixel** [comment, marked as answer]
Upgrade to the latest version, this is already fixed.

**Comment by mbroekman** [comment]
Hi. 

I'm running the latest version and got still the problem with the no show of the linked logo, also i have an access problem while using own (uploaded) images at the login page. The error is 403 ( forbidden). 

Do i need to do something special to make the uploaded images public ?

**Host Info**

OS: Docker Container (Linux)
Wiki.js version: 2.5.191
Database engine: PostgreSQL 11.11
Node version: 14.16

![image](https://user-images.githubusercontent.com/4540115/112300250-ed5aca80-8c98-11eb-93ba-c70e487102ac.png)
